sample-apps/discounts-tutorial/web/index.js

```diff
@@ -182,7 +278,7 @@ export async function createServer(
     const data = await client.query({
       data: {
         query: mutation,
-        variables: { discount: req.body },
```

Comment:
@andrewhassan I had to make this small tweak otherwise I would have to duplicate 99% of this function just because the other queries also require `id` as a variable.

Reply:
Yeah, I think that's a fair tradeoff.